source/_integrations/somfy_mylink.markdown

```diff
@@ -7,11 +7,16 @@ ha_category:
 ha_release: 0.92
 ha_iot_class: Assumed State
 ha_domain: somfy_mylink
+ha_config_flow: true
+ha_codeowners:
+  - '@bdraco'
 ---
 
 The `Somfy MyLink` integration is used as an interface to a compatible Somfy MyLink hub utilizing the `Synergy` API. It allows the addition of covers from the Somfy MyLink platform to Home Assistant.
 
-To use your compatible `Somfy MyLink` devices in your installation, add the following to your `configuration.yaml` file:
+To use your compatible `Somfy MyLink` devices in your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Somfy MyLink**.
+
+Alternatively, add the following to your `configuration.yaml` file:
```

Comment:
We should deprecate YAML based configurations and thus remove them from documentation (preventing people to start relying on them).

Reply:
Removed, and added a note about reversed devices.